<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navconnect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionContentsOffline"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionResetMessages"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOptions"/>
   <addaction name="actionContents"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/application-exit.svg</normaloff>:/littlenavconnect/resources/icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/navconnect.svg</normaloff>:/littlenavconnect/resources/icons/navconnect.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About Little Navconnect</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/qticon.png</normaloff>:/littlenavconnect/resources/icons/qticon.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/help.svg</normaloff>:/littlenavconnect/resources/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents (Online)</string>
   </property>
  </action>
  <action name="actionResetMessages">
   <property name="text">
    <string>&amp;Reset Messages</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/settings.svg</normaloff>:/littlenavconnect/resources/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Options ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+,</string>
   </property>
  </action>
  <action name="actionReplayFileSave">
   <property name="text">
    <string>&amp;Save Replay File ...</string>
   </property>
  </action>
  <action name="actionReplayFileLoad">
   <property name="text">
    <string>Open Replay &amp;File ...</string>
   </property>
   <property name="toolTip">
    <string>Open Replay File</string>
   </property>
  </action>
  <action name="actionReplayStop">
   <property name="text">
    <string>Stop Saving or Loading of Replay</string>
   </property>
   <property name="toolTip">
    <string>Stop Saving or Loading of Replay</string>
   </property>
  </action>
  <action name="actionContentsOffline">
   <property name="icon">
    <iconset resource="littlenavconnect.qrc">
     <normaloff>:/littlenavconnect/resources/icons/help.svg</normaloff>:/littlenavconnect/resources/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents (Offline, PDF)</string>
   </property>
  </action>
  <action name="actionConnectFsx">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FSX or Prepar3D</string>
   </property>
   <property name="toolTip">
    <string>Connect to FSX or Prepar3D using SimConnect.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionConnectXplane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>X-Plane</string>
   </property>
   <property name="toolTip">
    <string>Connect to X-Plane using the Little Xpconnect plugin.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="littlenavconnect.qrc"/>
 </resources>
 <connections/>
</ui>
